airbyte-webapp/src/core/request/AirbyteRequestService.ts

```diff
@@ -5,11 +5,18 @@ import { RequestMiddleware } from "./RequestMiddleware";
 import { VersionError } from "./VersionError";
 
 abstract class AirbyteRequestService {
-  constructor(protected rootUrl: string, private middlewares: RequestMiddleware[] = []) {}
+  constructor(private rootUrl: string, private middlewares: RequestMiddleware[] = []) {}
+
+  protected get requestOptions() {
+    return {
+      config: { apiUrl: this.rootUrl },
+      middlewares: this.middlewares,
+    };
+  }
 
   /** Perform network request */
   public async fetch<T = Response>(url: string, body?: unknown, options?: Partial<RequestInit>): Promise<T> {
-    const path = `${this.rootUrl}${url}`;
+    const path = `${this.rootUrl}/v1/${url}`;
```

Comment:
This requires updating `.env.development` to remove the `/v1/`

Reply:
I guess no trailing slash needed, then!

Reply:
I feel like _something_ is going to break from this. Just haven't found it yet. Well, more than the two...